skynet/modules/ttt/summaries/v1/router.py

```diff
@@ -21,12 +24,16 @@ async def get_action_items(payload: DocumentPayload) -> JobId:
 
 @api_version(1)
 @router.post("/summary")
-async def get_summary(payload: DocumentPayload) -> JobId:
+async def get_summary(payload: DocumentPayload, request: Request) -> JobId:
     """
     Starts a job to summarize the given payload.
     """
 
-    return await create_job(job_type=JobType.SUMMARY, payload=payload)
+    bearer_jwt: JWTBearer = await JWTBearer().__call__(request)
+    decoded_jwt = await authorize(bearer_jwt)
+    customer_id = decoded_jwt.get("cid")
```

Comment:
Let's catch potential problems and return a proper response.

Reply:
that is done through the raise statements from `authorize`. isnt that enough?

Reply:
That will return _some_ reply, but we can be more precise if we customize it, right? Like, the `cid` field is missing, etc.